src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +34,16 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        for (int i = 0; i < numbers.size(); i++) {
+            if (i % 2 != 0) {
+                numbers.set(i, numbers.get(i) - 1);
+            }
+        }
```

Comment:
```suggestion
        .map(index -> index % 2 != 0 ? numbers.get(index) - 1 : numbers.get(index))
```
Можливо для отримання индексів знадобиться метод [IntStream.range](https://hr-vector.com/java/metod-intstream-range)

Reply:
I made it clear for myself. Thanks for info and time